infrastructure/pom.xml

```diff
@@ -380,31 +92,6 @@
                     <shadedClassifierName>shaded</shadedClassifierName>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>com.github.os72</groupId>
-                <artifactId>protoc-jar-maven-plugin</artifactId>
```

Comment:
Does this mean protoc must be installed already in any environment that needs to build corfu? What if a build environment does not have protoc compiler present?

Reply:
nope, you can see that I have deleted a second copy of the plugin. The first one is still present in the pom.
So I just deleted an extra /duplicate plugin

Reply:
thanks